extensions/resource-deployment/src/constants.ts

```diff
@@ -6,3 +6,13 @@
 export const DeploymentConfigurationKey: string = 'deployment';
 export const AzdataInstallLocationKey: string = 'azdataInstallLocation';
 export const ToolsInstallPath = 'AZDATA_NB_VAR_TOOLS_INSTALLATION_PATH';
+
+export enum ResourceTypeCategories {
+	ALL = 'All',
+	ONPREM = 'On-premises',
+	SQLSERVER = 'SQL Server',
+	HYBRID = 'Hybrid',
+	POSTGRESQL = 'PostgreSQL',
```

Comment:
I have seen us use just 'Postgres' in most places in our product vs PostgreSQL, so double-confirm this string with the PMs.

Reply:
I have added the string from the mockups. Will confirm again with the PM